test/integrators/autodiff_events.jl

```diff
@@ -1,19 +1,32 @@
 using DiffEqSensitivity
 using OrdinaryDiffEq, Calculus, Test
+using Zygote
 
 function f(du,u,p,t)
-  du[1] = -p[1]
-  du[2] = p[2]
+  du[1] = u[2]
+  du[2] = -p[1]
```

Comment:
The Jacobian of the previous example was zero.  So I changed to the bouncing ball case.

Reply:
Sounds good.